README.md

```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
+[**stock corporations**](https://www.thebalancesmb.com/what-is-a-stock-corporation-398479). We’ll
+explain all those terms in this chapter.
+
+We begin with the basics of how companies and stock work in the United States.
+Understanding these details will help you learn the role of equity in your employment and
+its purpose in a company’s overall structure, as well as what you can do with your stock,
+how to decide what to do, and, of course, how to do it.
+
+If you think you know all about stock corporations, you can
+[jump ahead](#equity-compensation-basics) to compensation.
 
 ### What is a company?
 
 - 🄳 A **company** is a legal entity formed under corporate law for the purpose of conducting
   trade. In the United States,
   [several kinds of business entities](https://en.wikipedia.org/wiki/Types_of_business_entity#United_States)
-  may be formed, or [**incorporated**](https://en.wikipedia.org/wiki/Incorporation_(business)),
+  may be formed, or [**incorporated**](<https://en.wikipedia.org/wiki/Incorporation_(business)>,
   under the laws of a specific state.
-- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company that
-  aspires to grow quickly.
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
   Once a company is established in the market and successful for a while, it usually stops
   being called a startup.
-  - ☝️ The term *startup* is pretty informal.
-    It’s not a legal concept, and not everyone uses it consistently.
-    For our purposes, it’s best just to think of it as the young period of a fast-growing
-    company.
+  - ☝️ Unlike some of the legal terms we discuss, the term *startup* is informal, and not
+    everyone uses it consistently.
+    As distinguished investor [Paul Graham]\([https://en.wikipedia.org/wiki/Paul*Graham*(programmer](<https://en.wikipedia.org/wiki/Paul_Graham_(programmer>))
+    [emphasizes](http://www.paulgraham.com/growth.html)—and certainly this is true for our
+    purposes here—it’s best to think of a startup as any [early stage](#stages-of-a-startup)
+    company intending to grow quickly.
   - 🚧 Groundwork:
     What are good stats on how many people work in startups vs established companies?
 - Companies in the US include sole proprietorships, partnerships, limited liability
   companies (LLCs), S corporations, and C corporations.
   - 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the
-    choice of entity for technology startups, and equity compensation in LLCs is dramatically
-    different from equity compensation in corporations.
-    So we’ll focus on the one type of company most readers of this Guide will be concerned
-    with: The C corporation.
+    choice of entity for startups in technology and in non-tech sectors, and equity
```

Comment:
Still have a few issues with this (it's longer than it needs to be, and it also again makes it sound like this is a startup-only Guide).

Let's just tighten this up to keep it to the point:

This Guide [does not cover](#scope) equity compensation in LLCs, S corporations, partnerships, or sole proprietorships. Equity compensation in each of these kinds of business entities is significantly different than in C corporations.